theme/sass/components/_card-component.scss

```diff
@@ -27,6 +27,20 @@
             &--vertical {
                 flex-wrap: wrap;
                 flex-direction: row;
+
+                .card-wrapper {
+                    flex: 0 0 33.333333%;
+                    max-width: 33.333333%;
+                    margin-bottom: 0;
+                }
+            }
+        }
+
+        @include breakpoint($lg) {
+            &--horizontal {
+                .card-wrapper {
+                    max-width: 50%;
```

Comment:
The width of the teaser cards should not be set here, this should be set on the main card component container, it'll make it easer for you later.

Reply:
done